src/host_capabilities/crypto.rs

```diff
@@ -0,0 +1,41 @@
+use crate::host_capabilities::crypto_v1::{
+    CertificateVerificationRequest, CertificateVerificationResponse,
+};
+use anyhow::{anyhow, Result};
+use serde::{Deserialize, Serialize};
+
+/// A x509 certificate
+#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq)]
+pub struct Certificate {
+    /// Which encoding is used by the certificate
+    pub encoding: CertificateEncoding,
+    /// Actual certificate
+    pub data: Vec<u8>,
+}
```

Comment:
Missing adding here something similar to:

```suggestion
    pub attime: Option<picky::UtcDate>,
}
```